deploy-idsvr-azure-aks.sh

```diff
@@ -2,40 +2,38 @@
 set -eo pipefail
 
 display_help() {
-    echo -e "Usage: $(basename "$0") [-h | --help] [-i | --install]  [-d | --delete]  \n" >&2
-    echo "** DESCRIPTION **"
-    echo -e "This script can be used to deploy Curity Identity Server in Azure kubernetes cluster. \n"
-    echo -e "OPTIONS \n"
-    echo " --help      shows this help message and exit                                                                 "
-    echo " --install   creates a aks cluster & deploys curity identity server along with other components               "
-    echo " --start     starts up the environment                                                                        "
-    echo " --stop      shuts down the environment                                                                       "
-    echo " --delete    deletes the aks k8s cluster & identity server deployment                                         "
+  echo -e "Usage: $(basename "$0") [-h | --help] [-i | --install]  [-d | --delete]  \n" >&2
+  echo "** DESCRIPTION **"
+  echo -e "This script can be used to deploy Curity Identity Server in Azure kubernetes cluster. \n"
+  echo -e "OPTIONS \n"
+  echo " --help      shows this help message and exit                                                                 "
+  echo " --install   creates an aks cluster & deploys Curity Identity Server along with other components              "
+  echo " --start     starts up the environment                                                                        "
+  echo " --stop      shuts down the environment                                                                       "
+  echo " --delete    deletes the aks k8s cluster & Curity Identity Server deployment                                  "
```

Comment:
aks k8s cluster => AKS cluster ?

in the description of the install command, it says a "aks cluster" is created. Consequently, the delete command should remove such.